### DISCUSSION TITLE
Ross School Spiral Curriculum (designed by William Irwin Thompson)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#387] [Research]
Fascinating example of an "teal / integral" educational curricula (way ahead of its time). Interesting to know if other examples like this out there.

https://spiral.ross.org/

> The Ross Spiral Curriculum is a literary narrative of the evolution of human consciousness. It is taught chronologically through the grades as a dynamic system expanding in complexity. Cultural History is at the core of the Ross Curriculum, interwoven with all of the other domains. While each discipline offers its own rigorous curriculum, integration between domains creates a dynamic choreography of learning. Using this interdisciplinary approach, each grade focuses on a particular historical period and theme. The Spiral provides a cohesive architecture for analysis of the past as a dynamical system and enables students to more fully understand the present while envisioning the future.

Developed by legendary [William Irwin Thompson](https://lifeitself.org/notes/william%20irwin%20thompson) in 90s for Ross School (private school founded in 1991 by Courtney Ross wife of billionaire Stephen Ross).

![image](https://user-images.githubusercontent.com/180658/227589538-d544d3a4-2f50-4875-b305-8286ab0815d4.png)

![image](https://user-images.githubusercontent.com/180658/227511122-3977213a-70d6-43fa-9e07-8a7a1fdca96a.png)

Tags: teal-school-curriculum ontological-development-guide

## Next steps

- [ ] Extract curriculum content
  - [ ] spiral.ross.org whilst really cool suffers from "[shiny frontend syndrome](https://blog.okfn.org/2007/11/07/give-us-the-data-raw-and-give-it-to-us-now/)" a bit - e.g. no easy way to get a single list of e.g. all the units or key reading etc
- [ ] See how much it covers cultural evolution and ontological development and in what way

**Comment by Alex-Goodall** [comment]
Actually, you're right about the SFE. It shows the scope very nicely, but a collection of drop-down menus to create a database search would also be very useful.

They don't show any accredited institutions: so does that indicate a problem in replicating the idea?

**Comment by rufuspollock** [comment]
## Getting the data

A bit of browser network inspection reveals the data is actually stored in nice CSVs.

https://spiral.ross.org/ross_api/static/ecologies.csv

```
ecologies.csv
mentalities.csv
bifurcations.csv
grades.csv
domains.csv
courses.csv
units.csv
les.csv
integrations.csv
threads.csv
```

Next steps will be to boot a data repo/project and put these in.

**Reply by Alex-Goodall** [reply]
Nicely done! 

Not sure what a data repo/project is, but look forward to checking it out. I checked out a few files - some very good content.

**Comment by rufuspollock** [comment]
Ok, i've pushed a repo with work so far and a plan of work in it if anyone wants to dive in 😄

https://github.com/life-itself/spiral.ross.org
